rllib/algorithms/sac/torch/sac_torch_learner.py

```diff
@@ -260,7 +250,8 @@ def compute_loss_for_module(
         critic_loss = torch.mean(
             # TODO (simon): Introduce priority weights when episode buffer is ready.
             # batch[PRIO_WEIGHTS] *
-            torch.nn.HuberLoss(reduction="none", delta=1.0)(
+            batch["weights"]
```

Comment:
Why don't we use the `PRIO_WEIGHTS` constant anymore?

Reply:
Yeah I changed this b/c I was on the opinion that this is in general a weighted loss and `PRIO_WEIGHTS` is a bit misleading to me as these are actually importance sampling weights. But I am fine with both - it is simply a suggestion. 

We could also define a new constant like `WEIGHTS` that could be reused by any algorithm that weights the items in the loss.

Reply:
Got it, makes sense! Another option could be "loss_mask", which I think I'm already using in PPO. It's a weighting between 0.0 and 1.0, correct?